src/components/heading/heading.style.js

```diff
@@ -64,4 +61,71 @@ StyledHeadingIcon.defaultProps = {
   theme: baseTheme,
 };
 
-export { StyledHeadingIcon, StyledHeading };
+const StyledSubHeader = styled.div`
+  margin-top: 5px;
+`;
+
+StyledSubHeader.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledHeader = styled.div`
+  position: relative;
+`;
+
+StyledHeader.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledDivider = styled.hr`
+  width: inherit;
+  border: 0;
+  height: 1px;
+  background: ${({ theme }) => theme.hr.background};
+  margin: 15px 0 20px;
+`;
+
+StyledDivider.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledSeparator = styled.hr`
+  border-top: 2px solid ${({ theme }) => theme.palette.slateTint(80)};
+  margin: 10px 0 8px;
+  text-align: left;
+  width: 50px;
+`;
+
+StyledSeparator.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledHeaders = styled.div`
+  margin-left: ${({ back }) => (back ? "30px" : "")};
+`;
+
+const StyledHeaderLink = styled.div`
+  margin-top: -14px;
```

Comment:
Do we really need this? I'm against using negative margins where possible

Reply:
Without this line of the code that backlink arrow looks like this ⬇️ 

<img width="546" alt="Screenshot 2021-01-25 at 11 48 49" src="https://user-images.githubusercontent.com/56251247/105702189-532e0e80-5f03-11eb-9fa7-f78dc592910a.png">

So if it were to be removed, we'd need to find a way of correcting the position.

Reply:
Yeah that's what I meant really sorry, my comment was is there a better way of doing this positioning than using a negative margin? I haven't spend any time looking into it, but usually there is a better way

Reply:
changed display to flexbox